core/src/test/theme/theme.e2e.ts

```diff
@@ -22,20 +31,21 @@ const styleTestHelpers = `
     .ion-color-contrast {
       color: var(--ion-color-contrast);
     }
-    .ion-color-shade {
-      color: var(--ion-color-shade);
-    }
-    .ion-color-tint {
-      color: var(--ion-color-tint);
-    }
   </style>
 `;
 
 /**
- * All colors besides `light` should be tested against a dark background on dark theme.
+ * All colors should be tested in the following scenarios:
+ * 1) The base color as the text color against the contrast color as the background color
+ * 2) The contrast color as the text color against the base color as the background color
+ * 3) The contrast color as the text color against the shade color as the background color
+ * 4) The contrast color as the text color against the tint color as the background color
+ * 5) The base color as the text color against the base color at 0.08 opacity as the background color
+ * 6) The base color as the text color against the base color at 0.12 opacity as the background color
+ * 7) The base color as the text color against the base color at 0.16 opacity as the background color
  */
-configs({ modes: ['md', 'ios'], directions: ['ltr'], themes: ['dark'] }).forEach(({ config, title }) => {
-  const colors = ['primary', 'secondary', 'tertiary', 'success', 'warning', 'danger', 'medium', 'dark'];
+configs({ modes: ['md', 'ios'], directions: ['ltr'], themes: ['light', 'dark'] }).forEach(({ config, title }) => {
```

Comment:
_Maybe_. I believe the tests run against both modes because the `--ion-background-color` is set to `#121212` in `md` and `#000000` in `ios` for [dark theme](https://github.com/ionic-team/ionic-framework/blob/FW-5584/core/scripts/testing/themes/dark.css). But I have changed this test so that the text colors are on top of their contrast. I will talk with @sean-perkins to see if we need to adjust this.

Reply:
Spoke with Sean and decided it was safe to only use one mode now: https://github.com/ionic-team/ionic-framework/pull/28623/commits/6a1b4b59b3acb62dc7f7a145b17027a67247b1c1